app/assets/svgs/export.ts

```diff
@@ -1,4 +1,4 @@
 export default `<?xml version="1.0" encoding="UTF-8"?>
-<svg width="16" height="21" viewBox="0 0 16 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
what lead to this change?

Reply:
Color fill property was hardcoded with a specific color so we could not programmatically change it within the component. Also since this icon is not used anywhere except in the next steps screen so it seemed right to do this change now.